:sparkles:Feat blog CR 부분 models,url 연결완료

seoungjuyu opened the pull request:
## PR 타입(하나 이상의 PR 타입을 선택해주세요)
- [ ] 디자인 변경
- [x] 기능 추가
- [ ] 기능 삭제
- [ ] 버그 수정
- [ ] 의존성, 환경 변수, 빌드 관련 코드 업데이트

## 반영 브랜치
feature/1-blog-CR -> main

## 변경 사항
1. -주석처리된 부분은 아직 코드는 작성했으나 구현이 안된 코드입니다
-models 제시된 models로 작성해서 구현
-urls/views url이 연결되면 views 에서 해당 url을 작성하면 Http와 html 연결
2. 기능 1  메인화면
    - [ ] url연결

3. 기능 2 작성화면
    - [ ] url 연결



## 테스트 결과
변경 사항을 설명하는데 도움이 되는 스크린샷이나 참고 자료를 추가해주세요.
![화면 캡처 2022-10-25 213048](https://user-images.githubusercontent.com/110443976/197787693-97e754fe-01c4-4693-a1e4-2551adb9f81c.png)
main 연결
![화면 캡처 2022-10-25 213930](https://user-images.githubusercontent.com/110443976/197787726-303a9e8e-a391-4fc7-baeb-38304247164e.png)

작성폼 연결

LeeHyunji reviewed (changes requested):
전체적으로 제가 전달드린 요구사항이랑 많이 다릅니다.
개발에서 요구사항을 정확하게 파악하시는 것도 매우매우 중요합니다.

seoungjuyu committed:
:sparkles:Feat blog CR 부분 models,url 연결완료

-주석처리된 부분은 아직 코드는 작성했으나 구현이 안된 코드입니다
-models 제시된 models로 작성해서 구현
-urls/views url이 연결되면 views 에서 해당 url을 작성하면 Http와 html
연결
